assets/static/images/icon-ghost.svg

```diff
@@ -1 +1,6 @@
 d="M10 5.067l-.25-.002v.002H10zm0 5.138h-.25v.001H10zm-.967.7l.075-.238-.007-.002-.068.24zm-.07-.03l.13-.214-.13.214zm-1.3-.781l.128-.215-.125-.075-.127.072.123.218zM5.6 11.26l.122.218h.001L5.6 11.26zm-.158.04l.001-.25h-.005l.004.25zm-.158-.04l.123-.218-.123.218zm-2.065-1.167l.123-.218-.126-.072-.125.075.128.215zm-1.3.781l-.13-.214.13.214zm-.07.03l-.068-.24-.009.002.077.238zm-.661-.107l.148-.201-.003-.002-.145.203zm-.306-.593l.25.001h-.25zm8.497-.017l.25.002v-.002h-.25zm-.043.086l.138.208.007-.004.006-.005-.151-.2zm-7.817-.08h-.25v.003l.25-.003zm.043.086l-.15.2.006.004.006.004.138-.208zm.07.02l.01.25.064-.004.054-.032-.128-.215zm1.427-.857l-.127-.216-.002.002.128.214zm.164-.045l.001.25h.001l-.002-.25zm.159.04l.123-.217-.123.218zm2.068 1.17l-.123.217.123.07.123-.07-.123-.217zm2.067-1.171l.123.217.002-.001-.125-.216zm.323 0l.128-.215-.003-.002-.125.217zm1.427.856l-.13.214.055.033.064.003.01-.25zM5.45.75C2.82.75.63 2.662.63 5.067h.5c0-2.076 1.91-3.817 4.32-3.817v-.5zm4.799 4.32C10.268 2.661 8.078.75 5.45.75v.5c2.409 0 4.316 1.74 4.3 3.815l.5.004zm0 5.135V5.067h-.5v5.138h.5zm-.407.796a.979.979 0 00.407-.797l-.5.002a.48.48 0 01-.2.39l.293.405zm-.885.143A1 1 0 009.843 11l-.293-.405a.5.5 0 01-.442.07l-.15.478zm-.123-.055c.04.025.084.044.13.057l.136-.482a.036.036 0 01-.009-.003l-.257.428zm-1.301-.78l1.3.78.258-.428-1.3-.782-.258.43zm-1.81 1.17l2.061-1.168-.246-.435-2.062 1.167.246.435zm-.283.072a.572.572 0 00.281-.072l-.244-.437a.073.073 0 01-.035.01l-.002.5zm-.281-.073a.56.56 0 00.285.073l-.007-.5a.062.062 0 01-.03-.008l-.248.435zM3.096 10.31l2.065 1.167.246-.435-2.065-1.167-.246.435zm-1.049.778l1.301-.78-.257-.43-1.301.782.257.428zm-.13.057a.536.536 0 00.13-.057l-.258-.428a.037.037 0 01-.008.004l.136.48zm-.878-.147c.255.188.585.241.887.144l-.154-.476a.486.486 0 01-.436-.07l-.297.402zm-.407-.795a.975.975 0 00.41.798l.29-.408a.475.475 0 01-.2-.388l-.5-.002zm0-5.137v5.138h.5V5.067h-.5zm8.497 5.118c0-.044.022-.085.056-.11l.302.398a.359.359 0 00.142-.283l-.5-.005zm0-5.118v5.121h.5v-5.12h-.5zM5.45 1.892c2.064 0 3.68 1.454 3.68 3.175h.5c0-2.06-1.907-3.675-4.18-3.675v.5zM1.77 5.067c0-1.716 1.618-3.175 3.681-3.175v-.5c-2.273 0-4.181 1.62-4.181 3.675h.5zm0 5.127V5.067h-.5v5.127h.5zm-.056-.113a.141.141 0 01.056.11l-.5.006a.359.359 0 00.143.283l.301-.4zm-.092-.031a.134.134 0 01.08.022l-.277.416a.366.366 0 00.219.061l-.022-.5zm1.309-.821l-1.427.856.257.429 1.427-.857-.257-.428zm.292-.08a.579.579 0 00-.29.078l.253.432a.078.078 0 01.04-.01l-.003-.5zm.282.072a.568.568 0 00-.283-.073l.004.5c.012 0 .024.003.034.009l.245-.436zm2.07 1.169L3.503 9.22l-.245.435 2.068 1.17.246-.436zm1.82-1.17l-2.067 1.17.246.435 2.067-1.17-.247-.436zm.571 0a.572.572 0 00-.574 0l.251.433a.072.072 0 01.072 0l.251-.433zm1.43.858l-1.427-.856-.257.429 1.427.856.257-.428zm-.198-.012a.135.135 0 01.08-.023l-.021.5a.365.365 0 00.218-.061l-.277-.416z"/><ellipse cx="4.481" cy="5.881" rx=".674" ry=".649" stroke-width="0.25"/><ellipse cx="7.855" cy="5.881" rx=".674" ry=".649" stroke-width="0.25"/></svg>
```

Comment:
<GhostIcon/> was unused, so updating file to use ghost specified in the designs, which is filled in white.